roles/kubernetes-apps/ansible/defaults/main.yml

```diff
@@ -53,6 +57,7 @@ dashboard_cpu_limit: 100m
 dashboard_memory_limit: 256M
 dashboard_cpu_requests: 50m
 dashboard_memory_requests: 64M
+dashboard_deployment_nodeselector: "kubernetes.io/os: linux"
```

Comment:
ditto: I could not find any places this dashboard_deployment_nodeselector is used.

Reply:
I wasn't mindful. Nice catch :) Thank a lot!